### DISCUSSION TITLE
General chat

### DISCUSSION THREAD

**Discussion by C0rn3j** [C0rn3j/sc-controller#35] [General]
Discussion test

**Comment by sektorao** [comment]
Hey man, thanks for making this app. I'm using this on Mint (very noob) for racing games with Steam Controller and i have a question. I want to use only horizontal axis (yaw) for racing, but no matter what i try it always ends like i mapped the whole stick to that axis. So when i lean to side it turns, but it also moves to side if i tilt it up or down. I hope i explained it well enough. I map it trough "Per axis" option, and it's the same no matter if i assign only one axis or all. Anything i could do? Tnx.

**Reply by C0rn3j** [reply]
I forgot to respond to this, sorry - are you using a stick or a gyro as the input? 

Please try if it still happens with the latest version and also post the entire action command in here - feel free to open an Issue if you think it's a bug.

**Comment by nitram3** [comment]
I hope this is the right place to ask this…

Does this have the option to hide the actual controller, so apps only see the emulated one?
And, when I was still on Windows, I used DS4Windows which had an option to configure a dead zone + stick calibration, so the stick was centred. Back then I used an ~8 year old DS4, and the centre was quite off on the left stick.

So, I'm asking if it supports these features, and if not, whether they are planned.

**Reply by C0rn3j** [reply]
> Does this have the option to hide the actual controller, so apps only see the emulated one?

Over USB it works that way without intervention.

Over Bluetooth it takes over the inputs but the controller still technically shows up. I've seen someone work around it by setting `chmod 000` against the original input devices, which blocks new apps from accessing them while leaving the already-open SCC working, not sure if there's anything better than that, I might implement that as an option in the app in the future.

> configure a dead zone

Only possible for manually registered controllers at the moment.

You can implement deadzones per each action - which is arguably a bit unwieldy, and it is tied to the profile.

I will hopefully get to implementing it proper later - https://github.com/C0rn3j/sc-controller/issues/131

> stick calibration

Same as above, only possible for manually registered controllers.

That haven't really occurred to me, I've opened an issue to track that - https://github.com/C0rn3j/sc-controller/issues/165

**Reply by nitram3** [reply]
Thanks!
